examples/auctions/blind_auction.vy

```diff
@@ -1,4 +1,4 @@
-# Blind Auction # Adapted to Vyper from [Solidity by Example](https://github.com/ethereum/solidity/blob/develop/docs/solidity-by-example.rst#blind-auction-1)
+# Blind Auction. Ported to Vyper from [Solidity by Example](https://github.com/ethereum/solidity/blob/develop/docs/solidity-by-example.rst#blind-auction-1)
```

Comment:
i don't think we want to say ported, as the translation was done a long time ago and we don't want to guarantee that they do the same thing